### DISCUSSION TITLE
回顧日期：2024/11/29

### DISCUSSION THREAD

**Discussion by alvinlo62** [ByteEat8Bits/DevOps-Final-Project#38] [Announcements]
# 回顧上次衝刺結果，從中學習了甚麼？
**1. 哪些事做得不錯（What went well）？
2. 哪些事需要改進（What could be improved）？
3. 哪些事別再做了（What didn't work）？
4. 哪些事該啟動了（What shall we focus）？**
．．．

歡迎各位踴躍發表，也可以提出其他意見！

**Comment by alvinlo62** [comment]
1. 網頁框架的實作與部署、可以自動化抓取資料
2. 可能要在處理資料抓取遺漏的問題、K線圖的XY軸、圖表縮放功能、能不能查價?
3. 即時調API的想法好像行不通
4. 網頁的優化（外觀排版或是圖表樣式...）

**Reply by alvinlo62** [reply]
新聞區的添加，資料量太龐大了，所以這個Idea應該行不通？

**Comment by Weixun93** [comment]
1. 資料查找，研究各指標分析
2. X-axis of Candlestick Charts、查價規律性
3. 調用 API ，可能不能即時
4. 重新檢查資料檢索程式碼，判斷哪個邏輯比較好
